src/components/NavBar.tsx

```diff
@@ -241,9 +260,11 @@ function GetStarted() {
 const before_pages = ['', 'about', 'learn', 'solve']
 // TODO(SATH): After login navbar change
 // const after_pages = ['home', 'about', 'learn', 'solve', 'submit'];
-const settings = ['Profile', 'Groups', 'Logout']
+// const settings = ['Profile', 'Groups', 'Logout']
```

Comment:
should we be bringing in the after pages? why are we commenting out settings?

Reply:
She used an array of strings and map() to create that profile menu because the buttons had no functionality, but now they each have to have separate functionality. I guess I could just delete that array.

Reply:
I'm going to let Sath work on the after_pages, because its her plan and she already has progress on it.

Reply:
sounds good man